src/components/Layouts.jsx/MainLayout.jsx

```
@@ -941,6 +1104,9 @@ const MainLayout = (props) => {
                             padding: "0px 24px 0px 20px",
                           }}
                           onClick={() => {
+                            if (props.pageName === "wordsorimage" || "m5") {
```

Biome

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)